dynatrace-oneagent-operator/README.md

```diff
@@ -0,0 +1,93 @@
+Rolling out Dynatrace OneAgent via DaemonSet on a cluster is straightforward.
+Maintaining its lifecycle places a burden on the operational team.
+Dynatrace OneAgent Operator closes this gap by automating the repetitive steps involved in keeping Dynatrace OneAgent at its latest desired version.
+
+## Overview
+
+Dynatrace OneAgent Operator is based on [Operator SDK](https://github.com/operator-framework/operator-sdk) and uses its framework for interacting with Kubernetes and OpenShift environments.
+It watches custom resources `OneAgent` and monitors the desired state constantly.
+The rollout of Dynatrace OneAgent is managed by a DaemonSet initially.
+From here on Dynatrace OneAgent Operator controls the lifecycle and keeps track of new versions and triggers updates if required.
+
+![Overview](./overview.svg)
+
+
+## Supported platforms
+
+Depending of the version of the Dynatrace OneAgent Operator, it supports the following platforms:
+
+| Dynatrace OneAgent Operator Helm Chart version | Kubernetes | OpenShift Container Platform |
+| ---------------------------------------------- | ---------- | ---------------------------- |
+| v0.5.4                                         | 1.11+      | 3.11+                        |
+
+
+## Quick Start
+
+The Dynatrace OneAgent Operator acts on its separate namespace `dynatrace`.
+It holds the operator deployment and all dependent objects like permissions, custom resources and
+corresponding DaemonSets.
+To install the Dynatrace OneAgent Operator via Helm run the following command:
+
+### Adding Dynatrace OneAgent Helm repository
+```
+$ helm repo add dynatrace https://github.com/Dynatrace/helm-charts/repos/stable
+```
+
+
+#### Kubernetes
+```
+$ kubectl create namespace dynatrace
+$ helm install \
+    --name=dynatrace-oneagent-operator \
+    dynatrace/dynatrace-oneagent-operator \
+    -n dynatrace \
+    --set platform=kubernetes, \
+    oneagent.apiUrl=https://ENVIRONMENTID.live.dynatrace.com/api, \
+    secret.apiToken=DYNATRACE_API_TOKEN, \
+    secret.paasToken=PLATFORM_AS_A_SERVICE_TOKEN
+```
+
+#### OpenShift
+```
+$ oc adm new-project --node-selector="" dynatrace
+$ helm install \
+    --name=dynatrace-oneagent-operator \
+    dynatrace/dynatrace-oneagent-operator \
+    -n dynatrace \
+    --set platform=openshift, \
+    oneagent.apiUrl=https://ENVIRONMENTID.live.dynatrace.com/api, \
+    secret.apiToken=DYNATRACE_API_TOKEN, \
+    secret.paasToken=PLATFORM_AS_A_SERVICE_TOKEN
+```
+
+This will automatically install the Dynatrace OneAgent Operator and create OneAgents for every of your nodes.
+
+## Update procedure
+
+To update simply update your helm repositories and check the latest version
+
+```
+$ helm repo update
+$ helm search repo dynatrace-oneagent-operator
```

Comment:
What do you use `search` here for?

Reply:
Split the commands so the purpose of 'search' gets clearer